recipe/install_pip_metadata.patch

```diff
@@ -23,11 +23,7 @@ index c5df8bca8fc5..2acbe19b663c 100644
 +set(OPENCV_PYTHON_PIP_METADATA_INSTALLER "cmake" CACHE STRING "Specify the string to identify the pip Installer. Default: cmake, change this if you are using another tool.")
 +mark_as_advanced(OPENCV_PYTHON_PIP_METADATA_INSTALLER)
 +if (OPENCV_PYTHON_PIP_METADATA_INSTALL)
-+  if (WIN32)
-+    set(NEW_LINE "\n\r")
-+  else()
-+    set(NEW_LINE "\n")
-+  endif()
++  set(NEW_LINE "\n")
```

Comment:
if this works, can we remove the indirection all together? Can we type `\n` instead ?

Reply:
Done https://github.com/conda-forge/opencv-feedstock/pull/303/commits/8b045f77a69c2fd10b736fe5653f4636dd8483f9 .